docs/agave/guides/files/introduction.rst

```diff
@@ -581,7 +565,7 @@ Moving files and directories
 
 .. code-block:: plaintext
 
-   files-move -D $DESTPATH -S api.tacc.cloud $PATH
+   tapis files move AGAVE_URI DESTINATION
```

Comment:
See comment above, it is missing the comma. Please double-check if it is needed

Reply:
yes the comma is not needed.